kb-importer/src/test/java/org/eclipse/steady/kb/command/ImportTest.java

```diff
@@ -52,4 +60,18 @@ public void validationFail() throws ValidationException {
     args.put("d", "invalidDir");
     command.validate(args);
   }
+
+  @Test
+  public void testImportSkipExistingBug()
+      throws JsonSyntaxException, IOException, BackendConnectionException {
+    Vulnerability vuln = new Vulnerability();
+    vuln.setVulnId("CVE-TEST01");
+    MockBackConnector mockBackendConnector = new MockBackConnector();
+    HashMap<String, Object> args = new HashMap<String, Object>();
+    args.put("o", false);
```

Comment:
the arguments should be treated as properties with the same naming convention used in other modules and making use of org.eclipse.steady.shared.util.VulasConfiguration , e.g., vulas.kb-importer.overwrite

Reply:
They are currently defined as constants in the ImportCommand. Should I move it to the properties file or just use these constants?